R/quasi-label.R

```diff
@@ -66,6 +65,39 @@ quasi_capture <- function(.quo, .label, .capture, ...) {
   act
 }
 
+auto_label <- function(expr, value) {
+  if (is.call(expr) || is.name(expr)) {
+    label <- expr_label(expr)
+    if (can_inline(value)) {
+      paste0(label, " (", as_label(value), ")")
+    } else {
+      label
+    }
+  } else {
+    expr_label(expr)
```

Comment:
I'd avoid `expr_label()` as it's a remnant from lazyeval. Do you get good results if you use `as_label()` instead?